static/js/layout.js

```diff
@@ -181,6 +181,7 @@ function activateTabbarSwitcher(tabContainerId, tabBar, defaultTab) {
     }
 
     new bootstrap.Tab(someTabTriggerEl[0]).show()
+    $(`[href='#${selectedTab}']`).trigger("shown")
```

Comment:
This causes an infinite loop (on non firefox browsers) and therefor throwing an error in the console. 

I would suggest to remove it here and instead change the code in the slots.html line 292 to something like:
```      
var activateAttestations = ()=>{
  if(!attestations_tab_loaded) { setupInfiniteScrollAttestations(); }; attestations_tab_loaded = true;
 }
atab.on('shown.bs.tab', activateAttestations);
if (atab.hasClass("active")){
   activateAttestations()
}
 ```

Reply:
I don't know how I missed that loop, sorry and thanks.

I'll apply your suggestion, makes total sense :slightly_smiling_face: